canvas_modules/common-canvas/src/tooltip/tooltip.jsx

```diff
@@ -522,14 +540,17 @@ ToolTip.propTypes = {
 	showToolTipIfTruncated: PropTypes.bool, // Set to true to only display tooltip if full text does not fit in displayable width
 	truncatedRef: PropTypes.object,
 	delay: PropTypes.number,
-	showToolTipOnClick: PropTypes.bool
+	showToolTipOnClick: PropTypes.bool,
+	hoverable: PropTypes.bool, // if true, can hover over to the tooltip, instead of immedieatly disappearing.
```

Comment:
Typo - should be spelt: `immediately`  
Also, please start sentences with a capital letter. It's been a convention in English for several centuries, there's no need to change now.